spec/controllers/update_settings_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+  describe "Updating Settings" do
+    context "should log in" do
+    before do
```

Comment:
This can be cleaned up a bit.  Check out https://github.com/benwoody/shrtnr/blob/master/spec/support/authentication.rb#L7

for a feature helper to login a user.  Something like this would work:

```
before do
  login_as user
  visit settings_path
end

test test test
```

An `after` statement is run after the test is complete, so this test is "out of order"